pkg/coscheduling/coscheduling.go

```diff
@@ -71,10 +71,18 @@ func New(ctx context.Context, obj runtime.Object, handle framework.Handle) (fram
 	}
 
 	scheme := runtime.NewScheme()
-	_ = clientscheme.AddToScheme(scheme)
-	_ = v1.AddToScheme(scheme)
 	_ = v1alpha1.AddToScheme(scheme)
```

Comment:
why were the others `AddToScheme` removed?

Reply:
I don't find anywhere to call list or get or write to these resources.
And pods' schedule can still be completed after remove these AddToScheme.
Removing these AddToScheme can save memories because cache will not cache pods and nodes anymore.

Reply:
Which would be fine to me (if confirmed correct, haven't looked too deeply into this) but this seems unrelated to adding a cache; thus I'd move this specific change in its own commit if not its own pull request.

Reply:
Ok, I understood. I will move this to another pull request.